redirects.yaml

```diff
@@ -2,3 +2,4 @@ programmes/reseller/?: /programmes/channel
 amazon-web-services-llc/?: /aws
 programmes/phone/?: /
 hp/?: /hewlett-packard-enterprise
+programmes/openstack/?: /programmes
```

Comment:
Could it be set to deleted instead of redirecting to a page with no information about it?